tensorflow/python/ops/array_ops.py

```diff
@@ -3946,13 +3946,11 @@ def reverse_sequence(input,
   >>> input = [[1, 2, 3, 4, 5, 0, 0, 0], [1, 2, 0, 0, 0, 0, 0, 0],
   ... [1, 2, 3, 4, 0, 0, 0, 0], [1, 2, 3, 4, 5, 6, 7, 8]]
   >>> output = reverse_sequence(input, seq_lens, seq_dim=1, batch_dim=0)
```

Comment:
I think you need a print(output) here right?

Reply:
Yes, thanks @yashk2810. Updated.